src/OrbitBase/include/OrbitBase/TestUtils.h

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_BASE_TEST_UTILS_H_
+#define ORBIT_BASE_TEST_UTILS_H_
+
+#include <absl/strings/match.h>
+#include <absl/strings/str_cat.h>
+#include <absl/strings/str_format.h>
+#include <gmock/gmock.h>
+
+namespace orbit_base {
+
+namespace testing {
```

Comment:
we try to avoid nested namespaces. maybe ommit this, or rename orbit_base to orbit_base_testing?